src/components/pages/Triggers/TriggersList/TriggersList.tsx

```diff
@@ -0,0 +1,122 @@
+import {useContext, useEffect, useMemo, useState} from 'react';
+
+import {ExternalLink} from '@atoms';
+
+import {DashboardContext, MainContext} from '@contexts';
+
+import {Button, Modal, Skeleton, Text} from '@custom-antd';
+
+import {PageBlueprint} from '@organisms';
+
+import {Permissions, usePermission} from '@permissions/base';
+
+import {useGetTriggersKeyMapQuery, useGetTriggersListQuery} from '@services/triggers';
+
+import {useShallowGlobalStore} from '@store/GlobalStore';
+
+import {externalLinks} from '@utils/externalLinks';
+import {safeRefetch} from '@utils/fetchUtils';
+
+import AddTriggerModal from './AddTriggersModal';
+import EmptyTriggers from './EmptyTriggers';
+import {StyledTriggersGrid, StyledTriggersSkeletonWrapper, TriggerContainer} from './TriggersList.styled';
+
+const Triggers: React.FC = () => {
+  const {triggersList, setTriggersList, setTriggersKeyMap} = useShallowGlobalStore(state => ({
+    triggersList: state.triggersList,
+    setTriggersList: state.setTriggersList,
+    setTriggersKeyMap: state.setTriggersKeyMap,
+  }));
+
+  const {isClusterAvailable} = useContext(MainContext);
+  const {location, navigate} = useContext(DashboardContext);
+
+  const {data: triggers, refetch, isLoading} = useGetTriggersListQuery(null, {skip: !isClusterAvailable});
+  const {data: triggersKeyMap, refetch: refetchKeyMap} = useGetTriggersKeyMapQuery(null, {skip: !isClusterAvailable});
+
+  const [isAddTriggerModalVisible, setAddTriggerModalVisibility] = useState(false);
+  const mayCreate = usePermission(Permissions.createEntity);
+
+  const onNavigateToDetails = (name: string) => {
+    navigate(`/triggers/${name}`);
+  };
+
+  useEffect(() => {
+    if (triggers) {
+      setTriggersList(triggers);
+    }
+  }, [triggers]);
+
+  useEffect(() => {
+    if (triggersKeyMap) {
+      setTriggersKeyMap(triggersKeyMap);
+    }
+  }, [triggersKeyMap]);
```

Comment:
It should use `triggers` and `triggersKeyMap` instead

Reply:
What exactly and why?

Reply:
Basically,

* RTK returns now data to `triggers` / `triggersKeyMap` variables,
* These effects are moving these values (when not falsy) into `triggersList` / `triggersKeyMap`
* The whole code here is using `triggersList` / `triggersKeyMap` is unused

Instead, we could use here data directly from RTK, as these are recent.

Reply:
Ahh, actually `triggersKeyMap` is used, just in different places; additionally, it's downloaded and set also from multiple places though. I'm not sure what should be there, it will require much more complex analysis than what I did for now.

Reply:
It is basically options for triggers. User can come to list or details page and should have them available. Basically it is possible to fetch them as we fetch executors. But the difference here is that `triggersKeyMap` we use only in triggers, but executors could be present anywhere